src/billing-ui/List/ListPage.ts

```diff
@@ -0,0 +1,310 @@
+import _ from 'lodash/fp';
+import * as qs from 'qs';
+import { useEffect, useRef, useState } from 'react';
+import { div, h, h2, p, span } from 'react-hyperscript-helpers';
+import * as Auth from 'src/auth/auth';
+import { BillingProject, isCreating, isDeleting } from 'src/billing-logic/BillingProject';
+import { GoogleBillingAccount } from 'src/billing-logic/GoogleBillingAccount';
+import { billingRoles } from 'src/billing-logic/utils';
+import { CreateBillingProjectControl } from 'src/billing-ui/List/CreateBillingProjectControl';
+import { GCPNewBillingProjectModal } from 'src/billing-ui/List/GCPNewBillingProjectModal';
+import { ProjectListItem, ProjectListItemProps } from 'src/billing-ui/List/ProjectListItem';
+import { AzureBillingProjectWizard } from 'src/billing-ui/NewBillingProjectWizard/AzureBillingProjectWizard/AzureBillingProjectWizard';
+import { GCPBillingProjectWizard } from 'src/billing-ui/NewBillingProjectWizard/GCPBillingProjectWizard/GCPBillingProjectWizard';
+import ProjectDetail from 'src/billing-ui/Project';
+import Collapse from 'src/components/Collapse';
+import { customSpinnerOverlay } from 'src/components/common';
+import { Ajax } from 'src/libs/ajax';
+import colors from 'src/libs/colors';
+import { reportErrorAndRethrow } from 'src/libs/error';
+import Events from 'src/libs/events';
+import * as Nav from 'src/libs/nav';
+import { useCancellation, useOnMount } from 'src/libs/react-utils';
+import * as StateHistory from 'src/libs/state-history';
+import * as Style from 'src/libs/style';
+import * as Utils from 'src/libs/utils';
+import { useWorkspaces } from 'src/workspaces/common/state/useWorkspaces';
+import { CloudProvider, cloudProviderTypes } from 'src/workspaces/utils';
+
+const BillingProjectSubheader = ({ title, children }) =>
```

Comment:
Code came from https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/billing/List/List.ts, except that `TopBar`, `FooterWrapper`, and `navPaths` were removed.